src/middlewared/middlewared/plugins/chart_releases_linux/resources.py

```diff
@@ -14,6 +14,14 @@ class ChartReleaseService(Service):
     class Config:
         namespace = 'chart.release'
 
+    SYSTEM_CORES = None
```

Comment:
No reason to cache cpu core count since we're already caching it in another plugin. Please see `system.cpu_info`.

Reply:
Yes but that would mean making repeated calls i.e 20 apps and 20 redundant middleware calls to get the same attr in single `chart.release.query` call..